actors/common/src/main/java/org/sunbird/learner/actors/OrganisationManagementActor.java

```diff
@@ -478,17 +478,17 @@ private void createOrg(Request actorMessage) {
 
   private void createOrgExternalIdRecord(String channel, String externalId, String orgId) {
     Map<String, Object> orgExtIdRequest = new HashMap<String, Object>();
-    orgExtIdRequest.put(JsonKey.PROVIDER, channel);
-    orgExtIdRequest.put(JsonKey.EXTERNAL_ID, externalId);
+    orgExtIdRequest.put(JsonKey.PROVIDER, channel.toLowerCase());
+    orgExtIdRequest.put(JsonKey.EXTERNAL_ID, externalId.toLowerCase());
     orgExtIdRequest.put(JsonKey.ORG_ID, orgId);
 
     cassandraOperation.insertRecord(JsonKey.SUNBIRD, JsonKey.ORG_EXT_ID_DB, orgExtIdRequest);
   }
 
   private void deleteOrgExternalIdRecord(String channel, String externalId) {
     Map<String, String> orgExtIdRequest = new HashMap<String, String>();
-    orgExtIdRequest.put(JsonKey.PROVIDER, channel);
-    orgExtIdRequest.put(JsonKey.EXTERNAL_ID, externalId);
+    orgExtIdRequest.put(JsonKey.PROVIDER, channel.toLowerCase());
```

Comment:
same as above . and we can write common method will add provider and externalId in map and return map back, if every thing ok , same code used createOrgExternalIdRecord and  deleteOrgExternalIdRecord